cypress/e2e/gui/customers.cy.js

```diff
@@ -0,0 +1,187 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_URL = `${API_URL}/customers`;
+
+describe('Customers', () => {
+  describe('View', () => {
+    beforeEach(() => {
+      cy.setCookie('cookieConsent', 'accepted');     
+      cy.visit('/');
+    });
+
+    it('It goes back to the customers list when clicking the "Back" button', () => {
+      cy.get('button[aria-label^="View company:"]').first().click();
```

Comment:
Aqui vai uma sugestão mais simples, seguindo o que usou na linha abaixo:

```suggestion
      cy.contains('button', 'View').click();
```

Tal código é mais curto, direto-ao-ponto e possui um nível a menos na cadeia de comandos.

Reply:
Diferente do `cy.get()`, que retorna um ou mais elementos, o `cy.contains()` só retorna o primeiro elemento encontrado, evitando a necessidade do `.first()`.